From b05354a4a7f38e44f6d9da221576801ef5de6970 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Wed, 1 Sep 2010 19:04:36 +0000
Subject: [PATCH 09/13] gettextize: git-branch "(no branch)" message
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This is possibly a plumbing message.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---

ditto about the maybe plumbing output, but I think not.

 builtin/branch.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/builtin/branch.c b/builtin/branch.c
index 59e71fc..fc6c4c3 100644
--- a/builtin/branch.c
+++ b/builtin/branch.c
@@ -481,7 +481,7 @@ static void show_detached(struct ref_list *ref_list)
 
 	if (head_commit && is_descendant_of(head_commit, ref_list->with_commit)) {
 		struct ref_item item;
-		item.name = xstrdup("(no branch)");
+		item.name = xstrdup(_("(no branch)"));
 		item.len = strlen(item.name);
 		item.kind = REF_LOCAL_BRANCH;
 		item.dest = NULL;
-- 
1.7.2.2.579.g2183d

